ECoG_MAE/downstream_tasks/encoding/config.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass, field
+
+from config import ECoGDataConfig
+
+@dataclass
+class EncodingDataConfig(ECoGDataConfig):
+    # Path to the dataframe containing conversation data (i.e. embeddings, word onsets and offsets, etc)
+    conversation_data_df_path: str = ""
+    
+    # Path to parent directory of electrode data to be used in encoding.
+    encoding_neural_data_folder: str = ""
+    
+    # The glob path to access electrode data within encoding_neural_data_folder. Should include placeholder for electrode_id {elec_id} See default for sample value.
+    # TODO: In the future include options for subject id and conv id but keeping it simple for now with just electrode id.
+    electrode_glob_path: str = "NY*_*_Part*_conversation*_electrode_preprocess_file_{elec_id}.mat"
+
+    # Determines where relative to the word onset in ms the neural signal should be read from.
+    # Lag = 0 means neural signal will be read starting at word onset.
+    # Lag = -2000 means neural signal will be read starting at 2 seconds before word onset.
+    # Lag = 100 means neural signal will be read starting at 100 ms before word onset.
+    lag: int = 0
```

Comment:
In the lab's pipeline, models for each electrode are normally trained for a series of lags [-2000:2000]. We need to decide how to think about this here for benchmarking

Reply:
Good point, we can change this config to make this a list. I'm focusing on getting the model trained currently, so will add an issue for this to add support for this list.